Content.Server/SS220/TeleportationChasm/TeleportationChasmSystem.cs

```diff
@@ -31,18 +31,23 @@ public override void Update(float frameTime)//we cant teleport in shared, cause
         List<EntityUid> toRemove = [];
 
         var query = EntityQueryEnumerator<TeleportationChasmFallingComponent>();
-        while (query.MoveNext(out var uid, out var chasm))
+        while (query.MoveNext(out var uid, out var chasmFalling))
         {
-            if (_timing.CurTime < chasm.NextDeletionTime)
+            if (_timing.CurTime < chasmFalling.NextTeleportationTime)
                 continue;
 
-            TeleportToRandomLocation(uid);
+            if (chasmFalling.ShouldBeDeleted)
+            {
+                QueueDel(uid);
+                continue;
+            }
 
             toRemove.Add(uid);
```

Comment:
```suggestion
            toTeleport.Add(uid);
```
Переименуй тогда, потому что не понятно что ты ремуваешь, когда до этого уже QueueDel сделал